nutanix/resource_nutanix_project.go

```diff
@@ -306,7 +306,7 @@ func resourceNutanixProject() *schema.Resource {
 			"api_version": {
```

Comment:
I think this value should be computed and pass it hardoced in the APi requests similar than other resources.